site/tsconfig.json

```diff
@@ -23,8 +23,8 @@
       "@components/*": ["components/*"],
       "@commerce": ["../packages/commerce/src"],
       "@commerce/*": ["../packages/commerce/src/*"],
-      "@framework": ["../packages/local/src"],
-      "@framework/*": ["../packages/local/src/*"]
+      "@framework": ["../packages/sfcc/src"],
+      "@framework/*": ["../packages/sfcc/src/*"]
```

Comment:
```suggestion
      "@framework": ["../packages/local/src"],
      "@framework/*": ["../packages/local/src/*"]
```

Reply:
@lfades maybe we could add it to `.gitignore` and when doing changes in this file we could "force" add it

Reply:
reverted